psl-cli/src/main/java/org/linqs/psl/cli/Launcher.java

```diff
@@ -214,57 +204,16 @@ private Database runInference(Model model, DataStore dataStore, Set<StandardPred
         log.info("Inference Complete");
 
         // Output the results.
-        outputResults(database, dataStore, closedPredicates);
-
-        return database;
-    }
-
-    private void outputResults(Database database, DataStore dataStore, Set<StandardPredicate> closedPredicates) {
-        // Set of open predicates
-        Set<StandardPredicate> openPredicates = dataStore.getRegisteredPredicates();
-        openPredicates.removeAll(closedPredicates);
-
-        // If we are just writing to the console, use a more human-readable format.
-        if (!parsedOptions.hasOption(CommandLineLoader.OPTION_OUTPUT_DIR)) {
-            for (StandardPredicate openPredicate : openPredicates) {
-                for (GroundAtom atom : database.getAllGroundRandomVariableAtoms(openPredicate)) {
-                    System.out.println(atom.toString() + " = " + atom.getValue());
-                }
-            }
-
-            return;
+        if (!(parsedOptions.hasOption(CommandLineLoader.OPTION_OUTPUT_DIR))) {
+            log.trace("Writing inferred predicates to out stream.");
```

Comment:
Change the log level to info (so it's symmetric with the other logging statement).

Also, replace "out stream" with something like "stdout".